CHANGELOG.md

```diff
@@ -18,6 +18,9 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 
 - The `--offline-mode` flag for the `roxctl scanner generate` command was removed, as Scanner's default behavior is
   to fetch vulnerability updates from Central.
+- In version 4.0, RHACS released the collections feature that replaced access scopes used in report configurations.
```

Comment:
In 4.1, the notice made into the release notes but did not make it into the changelog. For posterity, could you add the changelog entry to 4.1?

https://docs.openshift.com/acs/4.1/release_notes/41-release-notes.html#notice-upcoming-report-change_release-notes-41